src/Converter/NoConverter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Kros.KORM.Converter
+{
+    internal class NoConverter : IConverter
+    {
+        public static IConverter Instance { get; } = new NoConverter();
```

Comment:
What about a private constructor? Force the use of default instance.

Reply:
Maybe. But the class is internal anyway.